src/runtime.js

```diff
@@ -57,7 +58,6 @@ class FetchEvent extends Event {
         ? readerFromStreamReader(resp.body.getReader())
         : undefined,
     });
-    return resp;
```

Comment:
👀 👍  ref: https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent/respondWith#return_value

Reply:
ah, this still returns Promise<void>, but according to the spec this should return undefined. Maybe we should stop using async function here.

Reply:
We need it to be async to await the response. https://github.com/denoland/deployctl/blob/534d3dde87a95c7b3ee3c0990a9f24d9618b4d5a/src/runtime.js#L60

Reply:
I think we can use `response.then` instead of await, but let's do that in another branch.